keyboards/ergodox_infinity/keymaps/rjhilgefort/keymap.c

```diff
@@ -0,0 +1,372 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#define BASEMAC 0 // Base Mac Layer
+#define BASELNX 1 // Base Linux Layer
+#define SYMB 2    // Symbols Layer
+#define MANMAC 3  // Media Numbers Mac Layer
+#define MANLNX 4  // Media Numbers Linux Layer
+#define EXA 5     // Extra Layer
```

Comment:
In general, we recommend an enum, since it is easier to maintain long term.
```suggestion
enum custom_layers {
    BASEMAC, // Base Mac Layer
    BASELNX, // Base Linux Layer
    SYMB,    // Symbols Layer
    MANMAC,  // Media Numbers Mac Layer
    MANLNX,  // Media Numbers Linux Layer
    EXA,     // Extra Layer
};
```

Reply:
Thanks for the tip, updated.